projects/plugins/jetpack/_inc/client/components/plans/plan-icon/index.jsx

```diff
@@ -194,5 +194,6 @@ PlanIcon.propTypes = {
 		PLAN_JETPACK_SECURITY_DAILY_MONTHLY,
 		PLAN_JETPACK_SECURITY_REALTIME,
 		PLAN_JETPACK_SECURITY_REALTIME_MONTHLY,
+		PLAN_JETPACK_VIDEOPRESS,
```

Comment:
Should `PLAN_JETPACK_VIDEOPRESS_MONTHLY` be on there as well?

Reply:
yes! bc8cbd98e3c7b4e1a7e618fcdf5d87c3a89d62e3